language/diem-framework/transaction_scripts/doc/transaction_script_documentation.md

```diff
@@ -852,7 +852,7 @@ and payee field being <code>child_address</code>. This is emitted on the new Chi
 
 <pre><code><b>include</b> <a href="../../modules/doc/DiemAccount.md#0x1_DiemAccount_TransactionChecks">DiemAccount::TransactionChecks</a>{sender: parent_vasp};
 <a name="create_child_vasp_account_parent_addr$1"></a>
-<b>let</b> parent_addr = <a href="../../modules/doc/Signer.md#0x1_Signer_spec_address_of">Signer::spec_address_of</a>(parent_vasp);
+<b>let</b> parent_addr = <a href="_spec_address_of">Signer::spec_address_of</a>(parent_vasp);
```

Comment:
Again this a bug. There is some code in docgen which tries to make a relative path [here](https://github.com/diem/diem/blob/4875f7ac1476ce8d84706be25f53e0b2eebb7bd2/language/move-prover/docgen/src/docgen.rs#L541) which I guess breaks either because of a bug or a precondition violation.

Reply:
Oh wait, so docgen actually does try to compute a relative path? Then it also looks like a bug to me. Anyways, for now let me try to pass in a relative path and we can fix docgen later.